lib/client.js

```diff
@@ -60,6 +60,16 @@ var clientOptions = require('./client-options');
  *  Setting it to false will cause less overhead and the reference of the network buffer to be maintained until the row / result set are de-referenced.
  *  Default: true.
  * </p>
+ * @property {Boolean} encoding.useUndefinedAsUnset Valid for Cassandra 2.2 and above. Determines that, if a parameter is set to
```

Comment:
Ah yes, good point.  I agree that we don't need validation on client initialization then.  The documentation is explicit and if a user encounters the error it is clear what the problem is.